x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/last_first_seen/index.ts

```diff
@@ -36,7 +36,9 @@ export const firstOrLastSeen: SecuritySolutionFactory<typeof FirstLastSeenQuery>
       dsl: [inspectStringifyObject(buildFirstOrLastSeenQuery(options))],
     };
 
-    if (options.order === 'asc') {
+    const { order } = parseOptions(options);
```

Comment:
Could you please add unit test here for cases where parsing fails/succeeds.

Additionally, when endpoint is called and parsing fails should it send a different status code? I am not sure but I think we should check.